webpack/assets/javascripts/react_app/components/CPUCoresInput/CPUCoresInput.js

```diff
@@ -36,38 +40,51 @@ const CPUCoresInput = ({
   const validationState = getValidationState();
 
   return (
-    <FormGroup validationState={validationState}>
-      <Grid>
-        <Row>
-          <Col>
-            <NumericInput
-              value={value}
-              minValue={minValue}
-              onChange={v => handleChange(v)}
-              label={label}
-            />
-          </Col>
-        </Row>
-        {validationState === 'warning' && (
-          <HelpBlock>
-            <WarningTriangleIcon className="help-block-icon" />
-            {__('Specified value is higher than recommended maximum')}
-          </HelpBlock>
-        )}
-        {validationState === 'error' && (
-          <HelpBlock>
-            <ErrorCircleOIcon className="help-block-icon" />
-            {__('Specified value is higher than maximum value')}
-          </HelpBlock>
-        )}
-      </Grid>
-    </FormGroup>
+    <Grid>
+      <Row>
+        <Col>
+          <NumericInput
+            value={value}
+            minValue={minValue}
+            onChange={v => handleChange(v)}
+            label={label}
+            disabled={disabled}
+            name={name}
+            id={id}
+            inputClassName={inputClassName}
+          />
+        </Col>
+      </Row>
+      <Row>
+        <Col className="col-md-2" />
+        <Col className="col-md-4">
```

Comment:
Also I think for the warnings it makes sense to take the rest of the Row so can you change 4 to 10?